auth0/setup-tenant.bash

```
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+source env.bash
+
+CLIENT_REDIRECT_URI=https://my.redirect.uri
```

Shellcheck (0.10.0)

[warning] CLIENT_REDIRECT_URI appears unused. Verify use (or export if used externally).

(SC2034)